peregrine/resources/submission/graphql/node.py

```diff
@@ -17,6 +17,8 @@
 import sqlalchemy as sa
 
 from datamodelutils import models as md  # noqa
+from promise import Promise
```

Comment:
can you add `promise` to the requirements.txt?

Reply:
sure